zerver/openapi/zulip.yaml

```diff
@@ -10767,6 +10767,50 @@ paths:
                         "result": "error",
                       }
                     description: An example JSON response for when the supplied stream does not exist
+  /streams/{stream_id}/delete_topic:
+    post:
+      operationId: delete-topic
+      summary: Delete a topic
+      tags: ["streams"]
+      description: |
+        Delete a topic given its name and its stream_id.
+
+        `POST {{ api_url }}/v1/streams/{stream_id}/delete_topic`
+      parameters:
+        - $ref: "#/components/parameters/StreamIdInPath"
+        - name: topic_name
+          in: query
+          description: |
+            The name of the topic whose messages should be marked as read.
```

Comment:
This text isn't accurate, copied from another endpoint.  It's important to re-review your own PRs to catch things like this.